hardpy/pytest_hardpy/utils/dialog_box.py

```diff
@@ -205,6 +207,71 @@ def convert_data(self, input_data: str | None = None) -> bool:
         return True
 
 
+class StepWidget(IWidget):
+    """Step widget.
+
+    Args:
+        title (str): Step title
+        text (str | None): Step text
+        widget (IWidget | None): Step widget
+
+    Raises:
+        WidgetInfoError: If the text or widget are not provided.
+    """
+
+    def __init__(self, title: str, text: str | None, widget: IWidget | None):
+        super().__init__(WidgetType.STEP)
+        if text is None and widget is None:
+            raise WidgetInfoError("Text or widget must be provided")
+        self.info["title"] = title
+        if isinstance(text, str):
+            self.info["text"] = text
+        if isinstance(widget, IWidget):
+            self.info["widget"] = widget.__dict__
+
+    def convert_data(self, input_data: str) -> None:
+        """Get the step widget data in the correct format.
+
+        Args:
+            input_data (str): input string
+
+        Returns:
+            None
+        """
+        pass
```

Comment:
Here and in `MultistepWidget` `return True` or `return None`. Docstring, as in `BaseWidget`.